torch/profiler/_pattern_matcher.py

```diff
@@ -263,7 +264,7 @@ def skip(self):
         # Anything less than sm_80 is not Ampere which doesn't support TF32
         has_tf32 = all(
             int(arch[3:]) >= 80 for arch in torch.cuda.get_arch_list())
-        return has_tf32 is False
+        return has_tf32 is False or super().skip or not self.prof.record_shapes
```

Comment:
Looks like there are several updates to prior PRs that you discovered here. You should move them to their respective PRs.